src/extension/hooks/useLocalStorage.jsx

```diff
@@ -16,7 +18,8 @@ export function useLocalStorage(key, initialValue) {
     try {
       const valueToStore = value instanceof Function ? value(storedValue) : value;
       setStoredValue(valueToStore);
-      window.localStorage.setItem(key, JSON.stringify(valueToStore));
+      let localStorageKey = `${STORAGE_KEY_PREFIX}${key}`;
+      window.localStorage.setItem(localStorageKey, JSON.stringify(valueToStore));
```

Comment:
looks like we need to prepend on the `getItem` call too on line 9?

Reply:
Good catch! Mb on that, added to the reader as well. 👍